beacon_node/network/src/sync/manager.rs

```diff
@@ -269,18 +258,11 @@ pub fn spawn<T: BeaconChainTypes>(
             beacon_chain.data_availability_checker.clone(),
             log.clone(),
         ),
-        delayed_lookups: delayed_lookups_send,
         log: log.clone(),
     };
 
     let log_clone = log.clone();
-    delayed_lookup::spawn_delayed_lookup_service(
-        &executor,
-        beacon_chain,
-        delayed_lookups_recv,
-        beacon_processor,
-        log,
-    );
+    delayed_lookup::spawn_delayed_lookup_service(&executor, beacon_chain, beacon_processor, log);
```

Comment:
I'm a little confused as to why we spawn this service in the sync manager. 
Currently, the way it is, we spawn the service in the sync manager, which calls a beacon processor method which used stuff from the beacon processor. Does it make sense to just spawn it in the beacon processor instead?

Reply:
good call ! moved it

Reply:
https://github.com/sigp/lighthouse/pull/4732/commits/1eec921f753e03407a92886ecf39460f67e01504